MediaBrowser.Model/Dlna/ProfileConditionValue.cs

```diff
@@ -27,6 +27,7 @@ public enum ProfileConditionValue
         IsInterlaced = 21,
         AudioSampleRate = 22,
         AudioBitDepth = 23,
-        VideoRangeType = 24
+        VideoRangeType = 24,
+        NumStreams = 25
```

Comment:
I think `StreamCount` is better than `NumStreams`, especially since the error is `StreamCountExceedsLimit`

Reply:
We have `Num*Streams` for audio and video:
https://github.com/jellyfin/jellyfin/blob/869d884e54cdce7340377064007c513a672ec314/MediaBrowser.Model/Dlna/ProfileConditionValue.cs#L22-L23